prow/cmd/checkconfig/main.go

```diff
@@ -266,15 +267,15 @@ func validateTideRequirements(cfg *config.Config, pcfg *plugins.Configuration) e
 		// using the matcher
 		config *orgRepoConfig
 	}{
-		{plugin: lgtm.PluginName, label: lgtm.LGTMLabel, matcher: requires},
-		{plugin: approve.PluginName, label: approve.ApprovedLabel, matcher: requires},
-		{plugin: hold.PluginName, label: hold.Label, matcher: forbids},
-		{plugin: wip.PluginName, label: wip.Label, matcher: forbids},
-		{plugin: verifyowners.PluginName, label: verifyowners.InvalidOwnersLabel, matcher: forbids},
+		{plugin: lgtm.PluginName, label: labels.LGTMLabel, matcher: requires},
```

Comment:
How about just `labels.LGTM`? Given the package name is `label`, adding a Label suffix to `labels.LGTMLabel` feels redundant, not entirely necessary and probably unneeded.